library/std/src/Std/OpenQASM/Intrinsic.qs

```diff
@@ -146,10 +167,37 @@ operation cx(ctrl : Qubit, qubit : Qubit) : Unit is Adj + Ctl {
     Std.Intrinsic.CNOT(ctrl, qubit);
 }
 
+operation cy(ctrl : Qubit, qubit : Qubit) : Unit is Adj + Ctl {
+    Controlled Std.Intrinsic.Y([ctrl], qubit);
```

Comment:
I'm wondering if CY and CZ from Canon namespace is any better. @swernli , what do you think?